src/__tests__/a11yAPI.test.js

```diff
@@ -27,13 +27,13 @@ class Button extends React.Component<any> {
       <TouchableOpacity
         accessibilityHint={BUTTON_HINT}
         accessibilityLabel={BUTTON_LABEL}
-        accessibilityRole={BUTTON_ROLE}
+        accessibilityRole={BUTTON_ROLE} // eslint-disable-line react-native-a11y/has-valid-accessibility-role
         accessibilityStates={['selected']}
       >
         <Typography
           accessibilityHint={TEXT_HINT}
           accessibilityLabel={TEXT_LABEL}
-          accessibilityRole={TEXT_ROLE}
+          accessibilityRole={TEXT_ROLE} // eslint-disable-line react-native-a11y/has-valid-accessibility-role
```

Comment:
TEXT_ROLE is set to link, so maybe it would be better to use it here directly and do not suppress eslint for a11y?

Reply:
Good point, it was only used in like 1 or 2 places, unnecessary